draft-ietf-tls-esni.md

```diff
@@ -155,21 +155,30 @@ not have access to the plaintext of the connection.
 
 ## SNI Encryption
 
-SNI encryption requires that each provider publish a public key and metadata which
-is used for SNI encryption for all the domains for which it serves directly or
-indirectly (via Split Mode). This document defines the format of the SNI encryption
-public key and metadata, referred to as an ESNI configuration, and delegates DNS
-publication details to {{!HTTPSSVC=I-D.nygren-dnsop-svcb-httpssvc}}, though other delivery
-mechanisms are possible. In particular, if some of the clients of a private
-server are applications rather than Web browsers, those applications might have
-the public key and metadata preconfigured.
+SNI encryption requires that each provider publish a public key and
+metadata which is used for SNI encryption for all the domains for
+which it serves directly or indirectly (via Split Mode). This document
+defines the format of the SNI encryption public key and metadata,
+referred to as an ESNI configuration, and delegates DNS publication
+details to {{!HTTPSSVC=I-D.nygren-dnsop-svcb-httpssvc}}, though other
+delivery mechanisms are possible. In particular, if some of the
+clients of a private server are applications rather than Web browsers,
```

Comment:
nit: Both the AP style guide and https://tools.ietf.org/html/rfc7322 no longer capitalize "web"